### DISCUSSION TITLE
Flare throw distance and sinking speed

### DISCUSSION THREAD

**Discussion by Kulik242** [FakeFishGames/Barotrauma#4821] [Ideas and suggestions]
Current flares are quite useful in enclosed spaces such as ruins and wrecks, but are next to useless in open spaces due to their fast sinking speed and the fact that when thrown they are flying off the screen.

The usefulness of the flare would be greatly increased if, when thrown, would fly somewhere around the viewing distance and sink much much slower than it does now. This way, it could be used to illuminate a direction from which monster are expected to swim.

**Comment by mLuby** [comment]
The use cases I can think of are:

1. Hold flare in your hand to omni-directionally light the room you're in or open ocean around you.
2. Throw flare into the corner of a dark room or cave to check for threats before you go there.
3. Drop flare on the floor to mark a location, designate a meeting point, or breadcrumb a path.

Are you suggesting there's also this use case? (which I've never encountered BTW.)

4. Drop or toss a flare so it falls slowly next to you as you swim.

**Reply by Kulik242** [reply]
As I've said there are useful in closed spaces since they remain on the floor providing illumination, in open seas, they just fall of the screen almost instantly.

**Comment by DeinococcusR** [comment]
I made a flare gun for one of my mods with exactly this open area use in mind. Shell drops slower, burns brighter but shorter and damages characters when holding in the hand.

**Comment by NilanthAnimosus** [comment]
Perhaps an alternative flare (One that uses the ever-abundant lead up i'd say or a little more plastic, or old flares can contain the lead etc and its of lighter materials) that has considerably less thrown power, and slight buoyancy to reduce how fast it sinks? I am also not sure it is possible to make a flare that has an increased drag to stabilize it better though to not float, and sink at a capped / steady speed to be useful enough. I recall some items dropped in very old past versions could sit exactly in place in the ocean without sinking or rising, though I doubt that is a thing anymore in the steam version of barotrauma.

I personally never use flares, and always take a flashlight. even in an enclosed space (Back when I used to use them) I found them just too dark and low radius compared to the inventory space they consumed to take with me compared to other equipment. I also find they get thrown off screen too easily as well, and in ocean for sure thrown well out of useful distances.

**Comment by Xeavas** [comment]
There's a new throwable light source in unstable called the Glowstick that floats in place when in water, which provides a good alternative to flares.

**Reply by Kulik242** [reply]
But do we need another item that serves almost the same purpose? What is the benefit of the flare over the glowstick, which would you bring on a mission?

**Reply by NilanthAnimosus** [reply]
I think from the (although light) description, it is a different item for intent and purpose, it may have a different radius and light strength, and in some cases you may want a light source to fall or be flung far such as where flares do work - ruins and wrecks. in the ocean however I imagine that glowstick works much better, but may stop too short in a ruin.

Since they've already added such - will have to think if it needs further discussion after seeing how it fares in usage thoughts on the glowstick and flare. and how they feel in different circumstances for usefulness.

**Reply by Xeavas** [reply]
With that glowsticks are better in terms of providing reliable lighting in a spot, which is great, and the flares sink rate can aid to distract creatures (they are sometimes attracted to it).

Edit : also different colours, I like variety :D
